src/constants/colors.ts

```diff
@@ -0,0 +1,5 @@
+export const COLORS = {
+  LIGHT_BLUE_0: '#EAF6FA',
```

Comment:
Just `LIGHT_BLUE` it's ok

Reply:
I call it like that because looking at the next assigments i can see more light blue variants along side that screen.